### ISSUE TITLE
Стварыць форму зваротнай сувязі

### ISSUE THREAD

**Issue by foksia07** [diglabby/falanster2020#26]
Як будзе вольная хвілінка, буду ўдзячная калі ты @fr0zen зможаш мне паказаць як гэтую штуковіну замуціць :)

**Comment by rizomaa** [comment]
@fr0zen мы планавали рабіць на старонцы кантактаў?

**Comment by fr0zen** [comment]
@rizomaa падаецца спачатку не планавалі. Экстра таск. Патрэбны сверкі дызайну. @foksia07 @svetit

**Comment by fr0zen** [comment]
#29

**Comment by rizomaa** [comment]
@fr0zen не трэба рабіць, калі гэта больш за 20 хвілін. @foksia07 казала
я замарожваю гэту картку, пакуль папярэднія не скончыш

**Comment by rizomaa** [comment]
@foksia07 дадай калі ласка проста мэйл. пакуль дастаткова.

**Comment by foksia07** [comment]
@дадала! Але, атрымоўліваецца - гэтую задачу не закрываем, покуль не даробім форму для зваротная сувязі?

**Comment by rizomaa** [comment]
@foksia07 так, але гэту картку пакуль можна пакінуць без выканаўца) дзякуй
